openpdf/src/main/java/com/lowagie/text/xml/xmp/XmpSchema.java

```diff
@@ -73,7 +73,8 @@ public XmpSchema(String xmlns) {
      * The String representation of the contents.
      * @return a String representation.
      */
-    public String toString() {
+    @Override
+    public synchronized String toString() {
```

Comment:
I’ve added the synchronized keyword here because the Hashmap toString method is and this method overrides it. So it should synchronized too. 
I should have mentioned it but this comes from LGTM. 
https://lgtm.com/projects/g/andreasrosdal/OpenPDF/snapshot/fa027397aba75245fee64c8622451f0dc2179dce/files/openpdf/src/main/java/com/lowagie/text/xml/xmp/XmpSchema.java?sort=name&dir=ASC&mode=heatmap#xe8460e01496d74f8:1

Reply:
Changing the StringBuffer to StringBuilder would make me change the `com.lowagie.text.xml.xmp.XmpSchema#process` protected method signature. Are you ok with it?

Reply:
Hm, how does this relate to changing access modifier?

Reply:
If the method is synchronized, then the StringBuffer (synchronized version of the StringBuilder) is not needed. and as we touch the method, maybe it is the moment to do it